.github/workflows/code_analysis.yaml

```diff
@@ -83,7 +88,7 @@ jobs:
             -
                 uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 8.1
+                    php-version: 8.2
```

Comment:
is it fine, that I have raised the php version for this analysis jobs?

Reply:
I want to strick with PHP 8.1 now, as I run it everywhere and Rector works well on it.

Reply:
done